ADRs/ADR0005-rename-fishermen.md

```diff
@@ -0,0 +1,55 @@
+# Rename Fishermen
```

Comment:
@dylanlott Do you want to share you discussion with the community?

Happy to do it on your behalf but think it's also a great way to reach out!

Reply:
I was trying to find this discussion but it seems that it has been removed. is it moving to the forum?